# Revert change to configure.in made between php 5.3.3 and 5.3.4 that disabled time_nanosleep on OS X.
--- a/configure.in.orig	2010-12-08 15:38:51.000000000 -0600
+++ b/configure.in	2010-07-21 05:50:07.000000000 -0500
@@ -611,7 +611,7 @@
 )
 
 dnl Some systems (like OpenSolaris) do not have nanosleep in libc
-PHP_CHECK_FUNC_LIB(nanosleep, rt)
+PHP_CHECK_FUNC(nanosleep, rt)
 
 dnl Check for getaddrinfo, should be a better way, but...
 dnl Also check for working getaddrinfo
